File: app/src/main/java/com/countrydelight/lintruleproject/java_test/HardCodedLiteralsTest.java
package com.countrydelight.lintruleproject.java_test;

import static kotlin.random.RandomKt.Random;

public class HardCodedLiteralsTest {

    // comment
    public void hardCodedNumberTest() {
        int a = 100;
        a = getA();
        boolean res = a == 100;
        if (a == 100) {
            a = 200;
        }
    }

    // comment
    private int getA() {
        return Random(100).nextInt();
    }

}

File: app/src/main/java/com/countrydelight/lintruleproject/kotlin_test/HardCodedLiteralsTest.kt
package com.countrydelight.lintruleproject.kotlin_test

import kotlin.random.Random

class HardCodedLiteralsTest {

    // comment
    fun hardCodedNumberTest() {
        var a = 100
        a = getA()
        val res = a == 100
        if (a == 100) {

        }
    }

    // comment
    private fun getA(): Int {
        return Random(100).nextInt()
    }
}